docs/source/backends.rst

```diff
@@ -31,24 +31,31 @@ torch.backends.cuda
 
 .. autofunction::  torch.backends.cuda.is_built
 
-.. attribute::  torch.backends.cuda.matmul.allow_tf32
+.. automodule:: torch.backends.cuda.matmul
```

Comment:
I think the problem is that this is not a module but a custom object.
Maybe use autoclass? Or do only torch.backends.cuda ?

Reply:
I use `.. currentmodule` to replace `automodule` and it works now.